doc/spec/ground_truth_init_parameters.adoc

```diff
@@ -5,17 +5,17 @@ endif::[]
 = Ground truth initialization parameters
 
 All models can optionally consume `osi3::GroundTruth` via an initialization parameter called `OSMPGroundTruthInit`.
-Its purpose is to provide the model with a view of the static environment, for example the map, in OSI format.
+Its purpose is to provide the model with information that does not change throughout the simulation, for example the map in OSI format or the model_reference of a MovingObject.
```

Comment:
```suggestion
Its purpose is to provide the model with information in OSI format that does not change throughout the simulation, i.e. is considered static.
This can encompass, for example, the road network or referenced 3D models.
```